src/docs/product/relay/best-practices.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Self-hosting - Best Practices
+sidebar_order: 5
+---
+
+This page contains some advices and best practices when self-hosting external Relays, i.e. Relays that run on your hardware and forward events to `sentry.io`.
+
+## General Considerations
+
+* While Relay can be built from source, the recommended way to run Relay is via the officially provided Docker image (`getsentry/relay`), which can be found on [DockerHub]. The images are tagged with their Git revision identifier.
+
+* We recommend to run at least two Relay instances (containers) with a reverse proxy (such as [HAProxy] or [Nginx]) in front of them for improved availability and to simplify Relay updates.
+
+* To monitor your Relay setup, you can configure [Logging] and [Metrics].
+
+## System Requirements and Recommendations
```

Comment:
I would build this out at a table, with the first column CPU, Memory, and Storage, and the second column our recommendation.

Reply:
Agreed. We can also have multiple columns for what we would consider low, medium, and high load profiles. For instance <100rps, <1000rps, >1000rps. All detail information, for instance the description of multi-threading can move into footnotes.

Things to consider adding in addition to CPU cores:
- clock speed
- instances
- network speed

What if we also add some of the example config values you mention below to this list?
- max_thread_count
- max_concurrent_requests
- event_buffer_size
- http.encoding

Reply:
Not super sure that we want network/clock speeds here, for various reasons.
Network: do we just say "have enough bandwidth"?
Clock speeds, CPU types, cache values: those vary too much and there's no linear correlation to make it useful.